lib/proxy/startup_info.rb

```diff
@@ -0,0 +1,16 @@
+require 'proxy/request'
+require 'socket'
+
+module Proxy
+  class StartupInfo < Proxy::HttpRequest::ForemanRequest
+    include Proxy::Log
+
+    def put_features
+      params = { :proxy_name => Socket.gethostname, :startup_refresh => true }.to_json
+      req = request_factory.create_put("/api/v2/smart_proxies/startup_refresh", params)
```

Comment:
Why using put here? This doesn't update a specific resource (no resource id in the url), a 'post' would be more appropriate, I think.